app/views.py

```diff
@@ -22,7 +19,7 @@ def info_mista():
     mista = db.session.query(OckovaciMisto.id, OckovaciMisto.nazev, Okres.nazev.label("okres"),
                              Kraj.nazev.label("kraj"), OckovaciMistoMetriky.registrace_fronta,
                              OckovaciMistoMetriky.registrace_prumer_cekani, OckovaciMistoMetriky.ockovani_odhad_cekani) \
-        .join(OckovaciMistoMetriky, (OckovaciMistoMetriky.id == OckovaciMisto.id)) \
+        .join(OckovaciMistoMetriky) \
```

Comment:
Maybe better to show which attribute is used...

Reply:
ETL crash if data are not present is intention, there should be separated task to improve ETL to be able to reconstruct historical data where possible (before registrations and reservations were available)